recipe/meta.yaml

```diff
@@ -6,36 +6,44 @@ package:
   version: {{ version }}
 
 source:
-  url: http://hepmc.web.cern.ch/hepmc/releases/{{ name }}-{{ version }}.tar.gz
-  sha256: 532c5387270dd55ff1e676e929a77c5910f2fba92fa62885b1f5ed0aef6fdde8
+  url: https://hepmc.web.cern.ch/hepmc/releases/{{ name }}-{{ version }}.tar.gz
+  sha256: 587faa6556cc54ccd89ad35421461b4761d7809bc17a2e72f5034daea142232b
   folder: source
+  patches:
+    # c.f. https://gitlab.cern.ch/hepmc/HepMC3/-/merge_requests/360
+    - 0001-fallback-to-readlink-if-macOS-missing-greadlink.patch  # [osx]
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
 
 requirements:
   build:
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - cmake
     - make
-  run_constrained:
-    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
```

Comment:
@chrisburr I think this is okay that we remove this? But maybe you can comment more.

Reply:
If it's working without setting anything in conda_build_config or playing with MACOSX_DEPLOYMENT_TARGET in the build script then it's all good.

See https://conda-forge.org/docs/maintainer/knowledge_base/#requiring-newer-macos-sdks